app/controllers/api/v0/manifests_controller.rb

```diff
@@ -12,30 +8,31 @@ def search
   end
 
   def validate
-    begin
-      manifest_content = JSON.parse(request.body.read)
+    manifest_content = read_body_as_json
+    unless performed?
```

Comment:
if `read_body_as_json` fails to parse JSON, we get a 400 error. That is tested here:

https://github.com/18F/e-manifest/blob/master/spec/requests/api/v0/validate_manifest_spec.rb#L4

In that case, we must check immediately after `read_body_as_json` is called in order to avoid double render.